### ISSUE TITLE
req.raw.cf types broken in 3.7.4

### ISSUE THREAD

**Issue by Cherry** [honojs/hono#1546]
### What version of Hono are you using?

3.7.4

### What runtime/platform is your app running on?

Cloudflare Workers

### What steps can reproduce the bug?

```javascript
import { Hono } from 'hono';

const app = new Hono();

app.get('/country', async (context) => {
	const country = context.req.raw?.cf?.country ?? '';
	context.text(`Country: ${country}`)
});

export default app;

```

### What is the expected behavior?

In version 3.7.3, `context.req.raw?.cf?.country ?? ''` does not have any issues. TypeScript is happy.

### What do you see instead?

In version 3.7.4, `context.req.raw?.cf?.country ?? ''`, the following TypeScript error is thrown:

```
src/index.ts:10:35 - error TS2339: Property 'cf' does not exist on type 'Request'.

10  const country = context.req.raw?.cf?.country ?? ''
```

**Comment by Cherry** [comment]
This appears to also be due to the `webworker` types included as reported in #1546.

**Comment by yusukebe** [comment]
@Cherry 
Thanks. I'll fix it soon.
